.github/workflows/sonar.yml

```diff
@@ -0,0 +1,131 @@
+name: "SonarCloud"
```

Comment:
Just curious about why you split the Sonar step? We may want to do it as well for other SR components.

Reply:
When it runs for a PR, this repo's Sonar secrets are not available to push the scan results.

Reply:
This allows it to work for both PRs and local repo pushes, although arguably PRs are more useful to have the scan.